trade-finance-manager-ui/templates/utilisation-reports/_macros/report-reconciliation-table.njk

```diff
@@ -2,56 +2,91 @@
 
 {% macro render(params) %}
   {% set reportReconciliationSummary = params.reportReconciliationSummary %}
+  {% set month = params.month %}
+  {% set year = params.year %}
+  {% set csrfToken = params.csrfToken %}
 
-  <table class="govuk-table" data-module="moj-sortable-table" data-cy="utilisation-report-reconciliation-table">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th scope="col" class="govuk-table__header" aria-sort="ascending">Bank</th>
-        <th scope="col" class="govuk-table__header" aria-sort="none">Status</th>
-        <th scope="col" class="govuk-table__header" aria-sort="none">Date report received</th>
-        <th scope="col" class="govuk-table__header" aria-sort="none">Total fees reported</th>
-        <th scope="col" class="govuk-table__header" aria-sort="none">Reported fees left to reconcile</th>
-        <th scope="col" class="govuk-table__header">Download report as CSV</th>
-      </tr>
-    </thead>
+  <form method="POST" data-cy="form">
```

Comment:
`method="PUT"` is not supported apparently, so this form needs to be submitted as a post request instead of put